nil deref panic on ctrl-L with no containers running

My local `docker` currently has no containers running; exploring `docui` on a fresh download ( 08d7aa1ad7 ) I pressed Ctrl-L and got a nil dereference panic.  

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x82e10f]

goroutine 84 [running]:
github.com/skanehira/docui/panel.(*ContainerList).ShowContainerLogs.func1(0xc000356380, 0xc0003904e0)
	/home/pdp/go/src/github.com/skanehira/docui/panel/containerPanel.go:716 +0x8f
created by github.com/skanehira/docui/panel.(*ContainerList).ShowContainerLogs
	/home/pdp/go/src/github.com/skanehira/docui/panel/containerPanel.go:708 +0x1e7
```

Compiled with `go1.13beta1`.  But really, "no containers, ask to view logs of container, get a panic".

--- a/panel/containerPanel.go
+++ b/panel/containerPanel.go
@@ -705,14 +705,14 @@ func (c *ContainerList) ShowContainerLogs() error {
 	signal.Notify(sigint, os.Interrupt)
 	errCh := make(chan error)
 
-	go func() {
-		selected, err := c.selected()
-		if err != nil {
-			c.ErrMessage(err.Error(), c.name)
-			common.Logger.Error(err)
-			errCh <- err
-		}
+	selected, err := c.selected()
+	if err != nil {
+		c.ErrMessage(err.Error(), c.name)
+		common.Logger.Error(err)
+		return err
+	}
 
+	go func() {
 		reader, err := c.Docker.ContainerLogStream(selected.ID)
 		if err != nil {
 			c.ErrMessage(err.Error(), c.name)

